saleor/graphql/account/filters.py

```diff
@@ -69,6 +69,13 @@ def filter_search(qs, _, value):
     return qs
 
 
+def filter_bot_search(qs, _, value):
+    search_fields = ("name", "is_active")
```

Comment:
Do we really want to match on `is_active`? Wouldn't it be better to have a boolean filter for it?